src/devices/bus/msx/cart/majutsushi.cpp

```diff
@@ -19,9 +19,9 @@ msx_cart_majutsushi_device::msx_cart_majutsushi_device(const machine_config &mco
 
 void msx_cart_majutsushi_device::device_add_mconfig(machine_config &config)
 {
-	// This is actually incorrect. The sound output is passed back into the MSX machine where it is mixed internally and output through the system 'speaker'.
-	SPEAKER(config, "speaker").front_center();
-	DAC_8BIT_R2R(config, m_dac, 0).add_route(ALL_OUTPUTS, "speaker", 0.05); // unknown DAC
+	DAC_8BIT_R2R(config, m_dac, 0);
+	if (parent_slot())
+		m_dac->add_route(ALL_OUTPUTS, soundin(), 0.15, AUTO_ALLOC_INPUT, 0); // unknown DAC
```

Comment:
The “unknown DAC” comment should go on the line with the DAC itself.

Reply:
moved the comment